srcCpp/PerftestTransport.cxx

```diff
@@ -712,6 +730,14 @@ void PerftestTransport::printTransportConfigurationSummary()
         stringStream << "\tNic: " << allowInterfaces << "\n";
     }
 
+    stringStream << "\tUse Multicast: " << ((allowsMulticast())? "True" : "False");
+    if(!allowsMulticast() && useMulticast){
+        stringStream << "  (Multicast is not supported for " 
+                     << transportConfig.nameString 
+                     << ")\n";
```

Comment:
<< ")\n"; 
It is not necessary, you have a "\n" in the next line.

Reply:
DONE